api/base/serializers.py

```diff
@@ -88,6 +89,26 @@ def to_internal_value(self, data):
         return super(JSONAPIListField, self).to_internal_value(data)
 
 
+class AuthorizedCharField(ser.CharField):
```

Comment:
That's the big problem with the SerializerMethodField. What if we start with this, and put a to-do in such that, if we need to make more like it, we do a composition instead?